src/main/java/core/basesyntax/SalaryInfoMain.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class SalaryInfoMain {
```

Comment:
no need to create this class - delete it

Reply:
not fixed